packages/notification-service/test/firebase.test.ts

```diff
@@ -23,6 +25,7 @@ const mockedMessagingSend = admin.messaging().send as jest.Mock
 
 describe('sendPaymentNotification', () => {
   beforeEach(() => {
+    _setKnownAddressesCache(new KnownAddressesCache())
```

Comment:
Maybe add a mock name/image here to test that it's actually set properly?

Reply:
Will do :)